### DISCUSSION TITLE
Expected timeline for a stable release?

### DISCUSSION THREAD

**Discussion by henduck** [openai/openai-dotnet#159] [General]
Super excited to finally be getting an _official_ OpenAI .NET SDK!!   What is the expected timeline for a stable release?  I have a product that uses one of the popular 3rd party .NET OpenAI wrapper libraries, but have encountered a few minor issues with that, and am trying to determine when this SDK is stable (or at least _stable enough_) to migrate to.

Thanks in advance!

**Comment by KrzysztofCwalina** [comment]
@mayurid, could you comment?

**Reply by henduck** [reply]
Just wanted to bump this back up... is there any ballpark on timeline for a stable release?   @mayurid ?

**Comment by mayurid** [comment]
Hi @henduck 
We really appreciate your patience here. We are continuing to work through all the service updates and customer feedback that we are receiving for the library. We will have a stable library this Fall. We will keep this thread updated as we come up with the exact time frames.

**Reply by henduck** [reply]
Thanks for the update @mayurid !

**Comment by joseharriaga** [comment]
Hello, @henduck . We released the first stable version of the library two days ago on September 30th. You can find it on NuGet:
🔗 https://www.nuget.org/packages/OpenAI/2.0.0

**Reply by henduck** [reply]
Excellent... looking forward to testing it out!   Thanks for the update @joseharriaga 👍
